docs/quickstart_guide.md

```diff
@@ -67,21 +73,52 @@ We tested the following instructions by setting up a **2-node** cluster on Cloud
 
 SSD-equipped nodes are highly recommended. Full list of CloudLab nodes can be found [here][cloudlab-hw].
 
+### 4. Go Installation
```

Comment:
this subsection is an unnecessary complication. we should just refer people to install the supported from the official instructions. Also, I don't think we support 1.20... can you check the dev guide where all verreions are listed and the CI to see the exact version?